pkg/codegen/docs/gen.go

```diff
@@ -1207,7 +1207,47 @@ func (mod *modContext) genIndex() indexData {
 	title := modName
 	menu := false
 	if title == "" {
-		title = mod.pkg.Name
+		// titleLookup is a map to map module package name to the desired display name
+		// for display in the TOC menu under API Reference.
+		titleLookup := map[string]string{
```

Comment:
Let's move this map to the top of the file, even though by the logic of `title` being empty, this will only be initialized once. It'll be easier to read this function.

Reply:
ok. cool. I moved it to the top where we delare the vars.